### DISCUSSION TITLE
Collapsible - close from inside

### DISCUSSION THREAD

**Discussion by dev-ton** [inkline/inkline#346] [Need help] [answered]

**Comment by alexgrozav** [comment]
Hey! Could you please add more details? What are you trying to achieve?

**Comment by dev-ton** [comment, marked as answer]
Sorry, I removed my message accidentally. Meanwhile I solved the problem already. I needed to close the collapsible using the button within the component, so I didi it by emiting v-model value with empty string and overwriting name of the collapsible item which is not in the list ('collapsible-item-2') and since I have just single item it worked well.

**Reply by alexgrozav** [reply]
Oh, I see. Happy to hear you solved it! 😃
